### PR TITLE
Add upgrade MTC-OADP integration upgrade steps

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**

### CODE REVIEW

File: docs/developer/MTC_OADP_Upgrade.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+<h1 align="center">MTC-OADP Integration Upgrade</h1>
+
+MTC 1.7 onwards, MTC will be depending on OADP for the provision of Velero and Restic. The following steps help in
+simulating the MTC-OADP Integration upgrade scenario via OLM.
+
+Prerequisites:
+- MTC 1.6.3 installed
+- OPM
+- export ORG="your ord name"
+- export TAF="your tag name"
```
Comment (rayfordj) [inline docs/developer/MTC_OADP_Upgrade.md]:
```suggestion
- export TAG="your tag name"
```

**Comment by weshayutin** [comment]
@shubham-pampattiwar is anyone automating the steps? Please ping me if there is interest in having it automated.
Thanks!

**Review by rayfordj — CHANGES_REQUESTED** [review]

File: docs/developer/MTC_OADP_Upgrade.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+spec:
+  channel: release-v1.7
+  installPlanApproval: Automatic
+  name: mtc-oadp-operator
+  source: mtc-oadp-operator
+  sourceNamespace: openshift-marketplace
+  startingCSV: mtc-operator.v1.6.3
+```
+Debugging tips:
+- Always stream the pod logs `catalog-oprator` pod from the `openshift-operator-lifecycle-manager` namespace when you edit
```
Comment (rayfordj) [inline docs/developer/MTC_OADP_Upgrade.md:187]:
```suggestion
- Always stream the pod logs `catalog-operator` pod from the `openshift-operator-lifecycle-manager` namespace when you edit
```

**Comment by shubham-pampattiwar** [comment]
> @shubham-pampattiwar is anyone automating the steps? Please ping me if there is interest in having it automated. Thanks!

@weshayutin Not that I am aware of. But yes seems like a good idea.

**Review by dymurray — APPROVED** [review]

**Comment by dymurray** [comment]
@rayfordj Please re-review when you get a minute. Thanks!

**Review by rayfordj — APPROVED** [review]
/lgtm

**Pull request merged**